src/pages/index.md

```diff
@@ -81,7 +81,15 @@ After defining your project, associating it with the Assurance API, and generati
 
 ![JSON Web Token](images/jwt_account.png)
 
-A JWT can be generated within the UI or using [several tools/libraries](https://jwt.io/libraries).
+A JWT can be generated within the UI or generated by code using several [tools/libraries](https://jwt.io/libraries). If the payload for signing is created using these tools, ensure the JWT payload is formatted as is shown in Developer Console:
```

Comment:
I'd just change it from the "several tools/libraries" to "several tools or libraries".